<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manageAct</class>
 <widget class="QMainWindow" name="manageAct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理活动</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	background-color: white;
	font: 12pt &quot;YaHei Monaco Hybird&quot;;
}
QPushButton{
	background-color: skyblue;
}
QLabel{
	font: 12pt &quot;YaHei Monaco Hybird&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateTimeEdit" name="end_dt">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="name_lab_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>60</horstretch>
      <verstretch>30</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>结束时间:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>90</y>
      <width>371</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="name_lab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>60</horstretch>
         <verstretch>30</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>当前活动:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="buddy">
        <cstring></cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name_le">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>200</horstretch>
         <verstretch>30</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>220</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>请输入活动名</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QDateTimeEdit" name="start_dt">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="name_lab_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>60</horstretch>
      <verstretch>30</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>开始时间:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>300</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="add_bt">
       <property name="text">
        <string>提交</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="return_bt">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="user_tb">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>541</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="output_btn">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>530</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>150</horstretch>
      <verstretch>40</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>导出签到表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addUser_btn">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>530</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>150</horstretch>
      <verstretch>40</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>添加签到人员</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>301</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>删除此活动</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
